packages/main/src/cache/persist.ts

```diff
@@ -1,4 +1,5 @@
 import imap from "imap"
+import { idFromHeaderValue } from "poodle-common/lib/models/uri"
```

Comment:
Message IDs are strings that come with angle brackets around them. But the brackets are not really part of the ID. `idFromHeaderValue` removes surrounding angle brackets from a string if there are any.